multihost_test/bz_automation/test_automation.py

```diff
@@ -347,3 +348,51 @@ def test_cve_access_control_bypass(self, multihost, bkp_pam_config, create_local
         result = multihost.master[0].run_command(f"sh /tmp/authentication_master.sh "
                                                  f"{multihost.client[0].sys_hostname}").stdout_text
         assert "Connection closed by" in result
+
+    @pytest.mark.tier1
+    def test_faillock_local_sudo_lockout(multihost, bkp_pam_config, create_user_with_sudo_access):
+        """
+        :title: Faillock feature configuration and behavior for local sudo user
```

Comment:
Please find a meaningful title for this test, one that explains the scenario that you are testing

Reply:
Done